dotnet/docfx.json

```diff
@@ -56,8 +56,8 @@
       "apiPlatform": "dotnet",
       "author": "natke",
       "ms.author": "nakersha",
+      "ms.date": "11/8/2019",
       "uhfHeaderId": "MSDocsHeader-DotNet",
-      "apiPlatform": "dotnet",
```

Comment:
why are you removing this? not sure what it does but it might be needed for .NET SDKs?

Reply:
Oh, yes, I should have noted that change too. That line was duplicated

Reply:
Ah, the other one was hidden 😄

Reply:
Yes, sorry about that